src/euphorie/client/config.py

```diff
@@ -1,3 +1,6 @@
 FULL_ACCOUNT = "full"
 GUEST_ACCOUNT = "guest"
 CONVERTED_ACCOUNT = "converted"
+
+LOCKING_SET_ACTIONS = ("lock_set", "lock_reset")
```

Comment:
Why `lock_reset`? “Reset” could be interpreted as the opposite of “set”. Additionally it does not match the method name `refresh_lock`.

I'd rather use `lock_refresh` or `lock_refreshed`.